### PR TITLE
[wasm] benchmark simple app without JSExport

### PR DESCRIPTION
**Pull request by pavelsavara**
The new benchmark `SimpleReachManaged` is comparable to `ReachManaged` but it doesn't contain any `JSExport`

Contributes to https://github.com/dotnet/runtime/issues/75598

### CODE REVIEW

File: src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj [hunk 1]
```diff
@@ -7,20 +7,38 @@
...
+  </ItemGroup>
 
   <ItemGroup>
     <WasmExtraFilesToDeploy Include="index.html" />
     <WasmExtraFilesToDeploy Include="appstart-frame.html" />
     <WasmExtraFilesToDeploy Include="frame-main.js" />
+    <WasmExtraFilesToDeploy Include="SimpleApp/appstart-simple.html" />
+    <WasmExtraFilesToDeploy Include="SimpleApp/simple-main.js" />
     <WasmExtraFilesToDeploy Include="style.css" />
     <Compile Remove="Console/Console.cs" />
```
Comment (radekdoulik) [inline src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj:26]: This line can be removed, as it is now removed in the item group above.

File: src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj [hunk 2]
```diff
@@ -7,20 +7,38 @@
...
+  </ItemGroup>
+
   <PropertyGroup>
     <_SampleProject>Wasm.Browser.Bench.Sample.csproj</_SampleProject>
     <_SampleAssembly> Wasm.Browser.Bench.Sample.dll</_SampleAssembly>
   </PropertyGroup>
 
   <Target Name="RunSample" DependsOnTargets="RunSampleWithBrowserAndSimpleServer" />
 
+  <ItemGroup>
```
Comment (radekdoulik) [inline src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj:40]: This item group is also not needed.

**Review by radekdoulik — APPROVED** [review]
Besides the small issues in [src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj](https://github.com/dotnet/runtime/pull/76632/files#diff-20ead8aa15e05c477369afd080456eb6a42a4518333b8eb68dc5a4431a393d21) it LGTM

File: src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj [hunk 3]
```diff
@@ -7,20 +7,38 @@
...
     <WasmExtraFilesToDeploy Include="appstart-frame.html" />
     <WasmExtraFilesToDeploy Include="frame-main.js" />
+    <WasmExtraFilesToDeploy Include="SimpleApp/appstart-simple.html" />
+    <WasmExtraFilesToDeploy Include="SimpleApp/simple-main.js" />
     <WasmExtraFilesToDeploy Include="style.css" />
     <Compile Remove="Console/Console.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="SimpleApp\Wasm.Browser.Bench.SimpleApp.csproj" />
```
Comment (radekdoulik) [inline src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj:30]: Shouldn't this have `ReferenceOutputAssembly="false"`?

**Comment by pavelsavara** [comment]
This is not good enough, because it still uses the same `mono-config.json` and therefore depends on size of the outer test.

**Pull request closed without merging**